src/components/Header.astro

```diff
@@ -0,0 +1,64 @@
+---
+import CentrapayLogo from './icons/CentrapayLogo.astro';
+import Bars from './icons/Bars.astro';
+import CloseOutline from './icons/CloseOutline.astro';
+import SiteSearch from './SiteSearch.vue';
+import MobileMenu from './MobileMenu.vue';
+
+const { path, navigation } = Astro.props;
+---
+<centrapay-header>
+  <header class="fixed bg-white inset-x-0 z-10 flex h-16 flex-shrink-0 shadow">
+    <div class="flex w-full justify-between items-center">
+      <div class="flex items-center">
+        <a
+          href="/"
+          class="flex items-center justify-center h-16 w-16 bg-brand-primary focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
+        >
+          <span class="sr-only">Go to home page</span>
+          <CentrapayLogo class="text-content-on-color icon-2xl" />
+        </a>
+        <div class="flex flex-row space-x-1 ml-7">
+          <a
+            href="/"
+            target="_self"
+            class="text-gray-600 text-sm leading-5 font-medium px-3 py-2 rounded-lg bg-gray-100 hover:bg-gray-200 hover:text-content-primary focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
+          >
+            Docs
+          </a>
+        </div>
+      </div>
+      <div class="flex items-center">
+        <SiteSearch client:load />
+        <MobileMenu client:load data-mobile-menu path={path} navigation={navigation} />
+      </div>
+    </div>
+  </header>
+</centrapay-header>
+
+<script>
+  class CentrapayHeader extends HTMLElement {
+    connectedCallback() {
+      const openBtn = this.querySelector('button[data-open-mobile-menu]');
+      const closeBtn = this.querySelector('button[data-close-mobile-menu]');
+      const mobileMenu = this.querySelector('[data-mobile-menu]');
+      if (!openBtn || !closeBtn || !mobileMenu) {
+        return;
+      }
+
+      openBtn.addEventListener('click', () => {
+        openBtn.style.display = 'none';
+        closeBtn.style.display = 'flex';
+        mobileMenu.style.display = 'relative';
+      });
+
+      closeBtn.addEventListener('click', () => {
```

Comment:
@MeganSteenkamp The mobile menu is broken on the preview site

1. Switch to mobile view
2. Open sidebar
3. Close sidebar - this doesn't work
4. Switching back to desktop view keeps the mobile menu open too

<img width="1434" alt="image" src="https://github.com/user-attachments/assets/4515ab64-c27d-4087-bc8c-c6514f8c89bd">

Reply:
@tom-blake this is already broken, see this comment: https://github.com/centrapay/centrapay-docs/pull/1080#discussion_r1837538743

The only difference is right now if you go back to desktop in the current site it does close. I don't see this as a user issue as you are normally on mobile or desktop, not switching between. I think we should fix this separately.